cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -283,6 +287,53 @@ data NodeHardForkProtocolConfiguration =
      }
   deriving (Eq, Show)
 
+-- | Find the starting era for the test network, if it was configured.
+--
+-- Starting eras have zero defined as a forking epoch. So here we're taking the last zeroed configuration value.
```

Comment:
Not necessarily. It doesn't have to be at epoch 0. For mainnet this will return `Nothing` because none of these config values are set but I expect that's what you want. You need to put a comment explaining this and include the PV2 issue. QA needs to test this and approve this PR to confirm this change presents no issues.